### ISSUE TITLE
Running tool results in 0's in each txt file

### ISSUE THREAD

**Issue by Limited55** [CYRiXplaysHalo/CarnageReporter#2]
When running the tool, it results in zeros being added into all files. I've attempted to run it multiple times and I dhd correctly change the .ini file to my gamertag.

I'm thinking its because you've used hardcoded filenames. Here is an example of my files:
![image](https://user-images.githubusercontent.com/13245395/85591119-edac3380-b63c-11ea-8341-94aba5f7fa4e.png)

Let me know if you need any more info.

**Comment by CYRiXplaysHalo** [comment]
So this is the folder where MCC generates the original files. The original files have these zeros at the end and I don't do anything to remove or alter them. All I do is add your username and the datetime info to the beginning of each filename to make it unique. So I think things are running correctly for you and there's nothing going wrong.

**Comment by Limited55** [comment]
Ah you are correct, I just opened one of the files and it doesn't track the data. Real shame, I'll raise a bug in flighting for it.

**Comment by Limited55** [comment]
> So this is the folder where MCC generates the original files. The original files have these zeros at the end and I don't do anything to remove or alter them. All I do is add your username and the datetime info to the beginning of each filename to make it unique. So I think things are running correctly for you and there's nothing going wrong.

So I just had another attempt at this and although the .XML files have kills, death counts etc. They never get added to the "total_deaths.txt" file, it remains 0.

I've tried with both the 0.5 source code, and the compiled .exe in the 0.4 release. My config.ini has been amended as per instructions and looks correct. Does it only work on new .xml files that are added following a completed game? I have existing an existing game I was hoping to start tracking. I assume I keep the .exe running in the background. Any ideas?

**Comment by CYRiXplaysHalo** [comment]
Yes it only works with new .xml files that are added after a game is completed. So once your start the application it will only pick up new games played. Just leave it on in the background while you play. It won't be able to go and retroactively calculate numbers for games that were played before the app was started. The main logic is once the app starts it looks to see if any one of the XML files has changed -- because MCC just overwrites the old file with the new one -- and once that has happened it will then read it in and crunch all of the numbers.
